### PR TITLE
Make nft widget collapsible

### LINKED ISSUE #4124: Collapsible NFTs component
Part of #nft rework

### PR DESCRIPTION
**Pull request by drillprop**
Fix #4124

### CODE REVIEW

**Comment by drillprop** [comment]
I allowed myself to do some cleaning on NFT Widget, to improve performance. I removed `content` in NftWidget and move it to the new component called NftWidgetContent

File: packages/atlas/src/components/_nft/NftWidget/NftWidgetContent.styles.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  align-items: center;
+  position: relative;
+  left: -4px;
+  z-index: 10;
+
+  &::before {
+    display: inline-block;
+    position: absolute;
+    content: '';
+    width: 28px;
```
Comment (attemka) [inline packages/atlas/src/components/_nft/NftWidget/NftWidgetContent.styles.ts:38]: why wont you use `sizes` here

File: packages/atlas/src/components/_nft/NftWidget/NftWidgetContent.styles.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+export const TopBidderContainer = styled.div`
+  display: flex;
+`
+
+export const TopBidderTokenContainer = styled.div<SizeProps>`
+  display: flex;
+  align-items: center;
+  position: relative;
+  left: -4px;
+  z-index: 10;
```
Comment (attemka) [inline packages/atlas/src/components/_nft/NftWidget/NftWidgetContent.styles.ts]: maybe worth using value from zIndexes list

**Review by attemka — CHANGES_REQUESTED** [review]

File: packages/atlas/src/components/_nft/NftWidget/NftWidgetContent.styles.ts (hunk 1 above)
Comment (drillprop) [reply packages/atlas/src/components/_nft/NftWidget/NftWidgetContent.styles.ts:38]: Sizes should be used for spacing, ie. margins, paddings, and gaps, not for fixed widths and heights. That was the primary usage for this util.

**Review by attemka — APPROVED** [review]

**Pull request merged**